system_tests/retry.py

```diff
@@ -0,0 +1,62 @@
+import time
+from functools import wraps
+
+import six
+
+
+class Retry(object):
+    """Retry class for retrying eventually consistent resources in testing."""
+
+    logger = None
+    exception = None
+    tries = None
+    delay = None
+    backoff = None
```

Comment:
All 5 of these class vars are overridden on instance creation, hence pointless here.